modules/ztp-configuring-spoke-at-remote-site.adoc

```diff
@@ -0,0 +1,162 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: PROCEDURE
+[id="configuring-edge-cluster-at-remote-location_{context}"]
+= Configuring the edge cluster at the remote location
+
+Configure the edge cluster by using the custom user interface.
+
+[NOTE]
+====
+Some of the commands need `root` access to run. You can either log in as the `root` user and proceed with the step or add `sudo` before every command.
+====
+
+.Prerequisites
+
+* `kubeadmin` password as supplied by vendor
+* URL of the custom user interface
+
+.Procedure
+
+. Unbox and turn on the cluster.
+
+. Log in at command line to `master-0` of the cluster:
+
+. Configure DNS on `master-0` of the hub cluster:
+
+.. Edit `resolv.conf` on `master-0` and add the IP address of `master-0`.
++
+[source,terminal]
+----
+$ vi /etc/resolv.conf
+----
+
+.. Add the line.
++
+[source,terminal]
+----
+nameserver 192.168.7.10
+----
+
+. Configure a static IP on the connected laptop:
+
+.. Determine the name of the laptop's network interface card (NIC) as follows.
++
+[source,terminal]
+----
+$ ip addr
+----
++
+[NOTE]
+====
+Look for the NIC name starting with the letter `e`.
+====
+
+.. Edit the NIC's network configuration.
++
+[source,terminal]
+----
+$ vi /etc/sysconfig/network-scripts/ifcfg-eth0
+----
++
+[NOTE]
+====
+Here `eth0` is the network card name, and it can be different for different computers.
+====
+
+.. Add or modify the configuration below:
++
+[source,terminal]
+----
+BOOTPROTO=static
+IPADDR=192.168.7.21
+NETMASK=255.255.255.0
+GATEWAY=192.168.0.1
+DNS1=8.8.8.8
+DNS2=8.8.4.4
+----
++
+[NOTE]
+====
+Use any IP in the range `192.168.7.20 - 192.168.7.150`.
+====
+
+. Restart the network services:
++
+[source,terminal]
+----
+$ systemctl restart NetworkManager
+----
+
+. Open a browser and log in to the edge cluster configuration user interface at the following URL with the supplied `kubeadmin` `username` and `password`:
++
+[source,terminal]
+----
+$ https://edge-cluster-setup.foobar-edge-cluster.domain.com
```

Comment:
This looks like the URL to go to, and not a command. So I'd drop the "$" at least.

Also, should it be clickable as is, or will the user have to modify it somehow? If it's clickable, I'd consider moving it out of the code block and allowing it to be clicked. However, I notice "foobar" in there, so I assume it can't. We technically shouldn't be using "foobar" in our docs - should that be replaced with "example", or some other thing in <> where it's obvious that they need to replace something?